test/http_server_spec/web/health-check.html

```diff
@@ -1,6 +1,14 @@
 <!doctype>
 <html lang="en-US">
   <head>
+    <!-- 
+      http_server_spec updated to reflect the fact that modern 
+      browsers will expect a link tag to exist in the head and 
+      will not load a CSS stylesheet when only referenced in 
+      the HTTP Link header. 
+      ref: https://http.dev/link 
+    -->
+    <link rel="stylesheet" href="layout-styles.css">
```

Comment:
This is not a [link header](https://http.dev/link), but is instead a [link element in the `<head>` tag section](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link), so I think this comment is incorrect.  Instead you can simply say something like this:
```suggestion
    <!-- 
      http_server_spec updated, because the external link to the css is not assumed
    -->
    <link rel="stylesheet" href="layout-styles.css">
```